src/source/PeerConnection/SessionDescription.c

```diff
@@ -398,7 +398,17 @@ STATUS populateSingleMediaSection(PKvsPeerConnection pKvsPeerConnection, PKvsRtp
     SPRINTF(pSdpMediaDescription->sdpAttributes[attributeCount].attributeValue, "%d", mediaSectionId);
     attributeCount++;
 
-    STRCPY(pSdpMediaDescription->sdpAttributes[attributeCount].attributeName, "sendrecv");
+    // Set appropriate directions in the SDP for the local transceivers
+    if(pKvsRtpTransceiver->transceiver.direction == RTC_RTP_TRANSCEIVER_DIRECTION_SENDRECV) {
```

Comment:
use switch/case instead of if/else if/else

Reply:
Done